aiohttp/payload.py

```diff
@@ -50,19 +61,26 @@ class PayloadRegistry:
     """
 
     def __init__(self):
-        self._registry = []
+        self._first = []
+        self._normal = []
+        self._last = []
 
-    def get(self, data, *args, **kwargs):
+    def get(self, data, _CHAIN=chain, *args, **kwargs):
         if isinstance(data, Payload):
             return data
-        for factory, type in self._registry:
+        for factory, type in _CHAIN(self._first, self._normal, self._last):
             if isinstance(data, type):
                 return factory(data, *args, **kwargs)
 
         raise LookupError()
 
-    def register(self, factory, type):
-        self._registry.append((factory, type))
+    def register(self, factory, type, order=Order.normal):
```

Comment:
Everywhere else `order` is keyword-only argument. Shouldn't it be there the same for consistency?

Reply:
fixed